src/solidity-frontend/solidity_convert.cpp

```diff
@@ -1596,6 +1605,18 @@ bool solidity_convertert::get_elementary_type_name(
     new_type.set("#cpp_type", c_type);
     break;
   }
+  case SolidityGrammar::ElementaryTypeNameT::STRING:
+  {
+    size_t value_length = 128;
+
+    new_type = array_typet(
```

Comment:
Add some comments to explain why array type is used? 
Are we converting "string" and "string memory" to  array type?